setup.py

```diff
@@ -26,7 +26,7 @@
     BuildJsSdkRegistryCommand,
 )
 
-VERSION = "10.0.0.dev0"
+VERSION = "10.0.0"
```

Comment:
I dunno how this is going to work, but master shouldn’t ever be 10.0.0, it needs a .dev0 or something. Since we’re not tagging or branching. Historically, I’d bump this in the release branch.